lib/transport/data_validation/aggregates/project.ex

```diff
@@ -64,12 +81,49 @@ defmodule Transport.DataValidation.Aggregates.Project do
     {:reply, {:ok, project}, project}
   end
 
+  def handle_call({:find_feed_source, %FindFeedSource{} = query}, _from, %__MODULE__{} = project) do
+    case handle_feed_source_action(project, query) do
+      {:ok, feed_source, project} -> {:reply, {:ok, feed_source}, project}
+      {:ok, feed_source} -> {:reply, {:ok, feed_source}, project}
+      {:error, error} -> {:reply, {:error, error}, project}
+    end
+  end
+
+  def handle_call({:create_feed_source, %CreateFeedSource{} = command}, _from, %__MODULE__{} = project) do
+    case handle_feed_source_action(project, command) do
+      {:ok, feed_source, project} -> {:reply, {:ok, feed_source}, project}
+      {:ok, feed_source} -> {:reply, {:ok, feed_source}, project}
+      {:error, error} -> {:reply, {:error, error}, project}
+    end
+  end
+
+  def handle_call({:validate_feed_source, %ValidateFeedSource{} = command}, _from, %__MODULE__{} = project) do
```

Comment:
What's the difference between a query and a command?

Reply:
A query retrieves data, and a command modifies it.